python/cudf/cudf/core/index.py

```diff
@@ -500,11 +499,11 @@ def memory_usage(self, deep=False):
         return self._values._memory_usage(deep=deep)
 
     @classmethod
-    def from_pandas(cls, index):
+    def from_pandas(cls, index, nan_as_null=None):
```

Comment:
Either way, it feels like we should either consistently default to `None` or consistently default to `True` across all of the cases of `from_pandas`.

Reply:
As earlier it didn't have nan_as_null parameter, I thought it would be better to follow `as_column` behavior. But it can be changed to True.

Reply:
> As earlier it didn't have nan_as_null parameter, I thought it would be better to follow `as_column` behavior. But it can be changed to True.

Following the behavior sounds good, so lets make everything default to `None` then 😄